src/components/views/rooms/MessageComposer.tsx

```diff
@@ -333,8 +332,12 @@ export class MessageComposer extends React.Component<IProps, IState> {
         this.messageComposerInput.current?.sendMessage();
 
         if (this.state.isWysiwygLabEnabled) {
+            // due to this function using wysiwyg functions targetting ES6,
+            // we need to import dynamically
+            const { sendMessage } = await import("./wysiwyg_composer/");
```

Comment:
FYI, we don't need to have a dynamic import for `sendMessage`, it doesn't have dependency with the wysiwyg lib